manual/sphinx/user_docs/boundary_options.rst

```diff
@@ -82,14 +84,185 @@ following are available:
+    bndry_ydown = fromFieldAligned(robin(1, 1, 0))
+
+.. _sec-parallel-bc-aligned-transform:
+
+Aligned transform boundary conditions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When using the 'aligned transform' method for parallel derivatives
+(see :ref:`sec-aligned-transform`), the way to apply parallel boundary
+conditions depends on how the method was implemented.
+
+For the 'implicit transform' version where the transformations to and
+from the field aligned grid are done within each parallel derivative
+or interpolation operator, the parallel boundary conditions must be
+applied to the base variable, so they must be applied using the
+``fromFieldAligned()`` modifier, as described in the previous section
+(:ref:`sec-parallel-bc-shifted-metric`).
+
+For the optimised method with separate objects for the field aligned
+versions of variables, it would be correct to apply boundary
+conditions using the ``fromFieldAligned()`` modifier before
+calculating the field aligned versions, but would add extra
+interpolations. Therefore the recommended way to apply parallel
+boundary conditions is to apply them directly to the field aligned
+versions of variables. Since the objects for the field aligned
+versions are not added to the time solver, it is necessary to load
+boundary conditions explicitly from the ``BOUT.inp`` input file during
+``PhysicsModel::init()``, for example by calling::
+
+    f_aligned.setBoundary("f_aligned")
+
+where the argument to ``setBoundary()`` specifies the name of the
+section in ``BOUT.inp`` from which boundary conditions will be read.
+Then the boundary conditions must be applied explicitly after the
+field aligned object has been calculated in ``PhysicsModel::rhs()``,
+for example::
+
+    f_aligned = fromFieldAligned(f);
+    f_aligned.applyBoundary();
+
+The boundary condition should be applied directly to the array in
+``f_aligned`` (not to parallel slices, which are not created for this
+scheme), so uses the 'standard' ``bndry_yup``/``bndry_ydown``. Radial
+boundary points should never be used from the aligned object, so its
+x-boundaries should be set to ``none``, and parallel boundary points
+should never be used from the base variable, so its y-boundaries
+should be set to ``none``. The input sections for ``f`` and
+``f_aligned`` might look like
+
+.. code-block:: cfg
+
+    [f]
+    bndry_xin = dirichlet
+    bndry_xout = dirichlet
+    bndry_yup = none
+    bndry_ydown = none
+
+    [f_aligned]
+    bndry_xin = none
+    bndry_xout = none
+    bndry_yup = free_o3
+    bndry_ydown = free_o3
+
+.. _sec-parallel-bc-fci:
+
+FCI boundary conditions
+^^^^^^^^^^^^^^^^^^^^^^^
+
+When using the FCI method (:ref:`sec-fci`), parallel boundary
+conditions must be applied to the parallel slices using
+`bndry_par_yup` and `bndry_par_ydown`, or `bndry_par_all` to set both
+together. The only option suitable for FCI currently implemented is
+``parallel_dirichlet_interp``. It is suggested, at least if there are
```

Comment:
Actually, I don't think this is correct, I thought that they were all suitable for FCI? Certainly when I originally wrote the parallel boundary conditions, they were all designed for FCI. Maybe @dschwoerer or @bshanahan have had more recent experience with them and can weigh in

Reply:
This is entirely my ignorance. `parallel_dirichlet`, `parallel_dirichlet_O3` and `parallel_neumann` looked to me like they should work for `ShiftedMetric`, and I just assumed `FCI` would need something special!

Someone who knows how `FCI` works - please edit this to say which boundary conditions make sense and when!

Reply:
I think they all should work. At least `parallel_neumann` is used a lot in hermes-2.

I have removed the sentence and added a bit FCI specific things.